pyfar/dsp/dsp.py

```diff
@@ -1285,7 +1285,7 @@ def find_impulse_response_delay(impulse_response, N=1):
             start_sample = np.nan
             warnings.warn(f"Starting sample not found for channel {ch}")
 
-        start_samples[ch] = start_sample
+        start_samples[ch] = np.squeeze(start_sample)
```

Comment:
```suggestion
        roots = np.squeeze(np.roots(search_region_poly))
        start_samples[ch] = start_sample
```
i would add it here, because here it starts, if I get the logic right, it the result should be squeezed, if all goes right.